lib/vdsm/tool/configurators/multipath.py

```diff
@@ -372,15 +370,13 @@ def isconfigured():
     """
     Check the multipath daemon configuration at /etc/multipath.conf.
     """
-
-    if os.path.exists(_CONF_FILE):
+    if os.path.exists(_CONF_FILE) and os.stat(_CONF_FILE).st_size > 0:
```

Comment:
Empty /etc/multipath.conf is not a valid configuration and we should not keep special code to handle it.

The file cannot be empty, and must contain the "RHEV REVISION X.Y" and "RHEV PRIVATE"
lines. If not the configuration is invalid and vdsm should not start!

Reply:
And it does. If the file exists **and has contents**, we verify contents, otherwise, we fail.

The only thing that changed is the "and" part.